doc/news/changes/minor/20171118GiovanniAlzetta-LucaHeltai

```diff
@@ -0,0 +1,5 @@
+New: Utilities::MPI::internal::all_gather and 
```

Comment:
What is the reason to make them internal? A user might want to use those too, don't you think?

Reply:
I just kept what Giovanni used at the beginning. I agree, I'll change that.